README.md

```diff
@@ -1,72 +1,40 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# capstone project
 
+> In this capstone project I managed to build the website based on an online website for a conference. using the provided design guidelines to create the website, I personalized the content.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS
+- JAVASCRIPT
 
 ## Live Demo (if available)
 
 [Live Demo Link](https://livedemo.com)
```

Comment:
- [ ] Please add a proper live demo link as per Microverse requirements. This would help anyone visiting your repo find what you actually did in your project. You can also share that link on your portfolio.